### PR TITLE
EASY-930

### PR DESCRIPTION
**Pull request by janvanmansum**
Fixed https://drivenbydata.atlassian.net/browse/EASY-930
@PaulBoon @GeorgiKhomeriki please review.

### CODE REVIEW

File: src/main/assembly/dist/cfg/logback.xml [hunk 1]
```diff
@@ -17,5 +17,7 @@
     <root level="error">
         <appender-ref ref="CONSOLE" />
     </root>
-    <logger name="nl.knaw.dans.easy.valchecksum" level="info" />
+    <logger name="nl.knaw.dans.easy.valchecksum" level="info" >
```
Comment (janvanmansum) [inline src/main/assembly/dist/cfg/logback.xml]: The EMAIL appender should send the log statements in an e-mail to configured recipients. Forgot to reference the appender earlier, so I fixed that here. I haven't been able to test this yet, as it doesn't seem to work from deasy.

File: src/main/scala/nl/knaw/dans/easy/valchecksum/FixityClient.scala [hunk 2]
```diff
@@ -50,11 +49,15 @@ class FixityClient(
...
-    mustSkipStream(pid, dsId) match {
-      case Success(true) => log.info(s"Skipping external datastream $pid/$dsId")
-      case Success(false) => validateDatastreamWithDelay(pid, dsId)
-      case Failure(e) => log.error(s"Could not get control group of $pid/$dsId: ${e.getMessage}")
+  def validateDatastream(pid: String)(dsSpec: DatastreamSpec): Unit =
+    mustSkipStream(pid, dsSpec.dsId) match {
+      case Success(true) => log.info(s"Skipping external datastream $pid/${dsSpec.dsId}")
+      case Success(false) => validateDatastreamWithDelay(pid, dsSpec.dsId)
+      case Failure(e) => e match {
+        case e: FedoraClientException =>
```
Comment (GeorgiKhomeriki) [inline src/main/scala/nl/knaw/dans/easy/valchecksum/FixityClient.scala]: Is this the only type of `Exception` that can occur here?
If not, then it's probably better to also add a wildcard clause for other errors.

Comment (GeorgiKhomeriki) [inline src/main/scala/nl/knaw/dans/easy/valchecksum/FixityClient.scala]: And you don't need the nested pattern matching here, you can match directly in the outer Failure clause on the specific type of the exception:

```Scala
case Failure(e: FedoraClientException) => // do logging
```

File: src/main/scala/nl/knaw/dans/easy/valchecksum/CLI.scala [hunk 3]
```diff
@@ -33,7 +34,10 @@ object CLI {
 
   private def getNamespaceSpecs: List[NamespaceSpec] = {
     val props = new PropertiesConfiguration(new File(homedir, "cfg/validation.properties"))
-    props.getKeys.map(ns =>
-      NamespaceSpec(ns, props.getStringArray(ns).toList)).toList
+    props.getKeys.map(ns => NamespaceSpec(ns, props.getStringArray(ns).toList.map(getDatastreamSpec))).toList
   }
+
+  private def getDatastreamSpec(s: String): DatastreamSpec =
+    if(s.endsWith("*")) DatastreamSpec(s.take(s.length - 1), true)
```
Comment (GeorgiKhomeriki) [inline src/main/scala/nl/knaw/dans/easy/valchecksum/CLI.scala]: You can also write `s.take(s.length - 1)` as `s.dropRight(1)`.

Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/valchecksum/CLI.scala]: Good idea!

**Pull request merged**